sphinx/environment/__init__.py

```diff
@@ -240,12 +242,20 @@ def _update_config(self, config):
             for item in config.filter('env'):
                 if self.config[item.name] != item.value:
                     self.config_status = CONFIG_CHANGED
+                    self.config_status_extra = __(' (%r)') % (item.name,)
                     break
 
             # this value is not covered by the above loop because it is handled
             # specially by the config class
             if self.config.extensions != config.extensions:
                 self.config_status = CONFIG_EXTENSIONS_CHANGED
+                diff = sorted(
+                    set(self.config.extensions) ^ set(config.extensions))
+                if len(diff) == 1:
+                    diff = diff[0]
+                else:
+                    diff = '%d' % (len(diff),)
+                self.config_status_extra = __(' (%r)' % (diff,))
```

Comment:
If you need to translate this, please wrap only format string by `__(...)` function? Our tooling extracts i18n strings from this declaration. So it would be better to surround only format string by `__(...)` function simply.